hack/generate.sh

```diff
@@ -39,6 +40,15 @@ AVAILABLE_CODEGEN_OPTIONS=(
   "provider_local"
   "extensions_config"
 )
+DEFAULT_MANIFESTS_FOLDERS=(
```

Comment:
What is the reason to call these dirs "manifest" dirs? Aren't these dirs simply ones that contain files with `go:generate` directive? Why we call them manifest dirs? For example in `cmd` or `extensions` I rather expect mocks to be generated, not some manifests.